modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/WishListItemResourceTest.java

```diff
@@ -76,24 +65,34 @@ protected String[] getAdditionalAssertFieldNames() {
 
 	@Override
 	protected WishListItem randomPatchWishListItem() throws Exception {
-		CProduct commerceProduct = _commerceCPDefinition2.getCProduct();
+		_cpInstance = CPTestUtil.addCPInstanceWithRandomSku(
```

Comment:
is now this method still needed? It's doing the same as the `randomWishListItem()`